index.html

```diff
@@ -1,20 +1,38 @@
 <!DOCTYPE html>
-<html lang="pl">
-<head>
-    <meta charset="UTF-8"/>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
-    <meta http-equiv="X-UA-Compatible" content="ie=edge"/>
-    <title>CodersCamp2020 | Star Wars API QUIZ</title>
-    <link rel="stylesheet" href="styles/App.css"/>
+<html lang="en">
 
-    <link href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
-          rel="stylesheet">
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CodersCamp2021 | Harry potter API QUIZ</title>
 </head>
+
 <body>
+    <header>
+        <div class="header-content">
+            <div class="header-content__game-logo"></div>
+            <div class="header-content__game-nav"> </div>
+        </div>
+    </header>
 
-<div id="swquiz-app">
-</div>
+    <aside>
+        <div class="game-image"></div>
+    </aside>
 
-<script src="src/index.js"></script>
+    <main>
+        <div class="game-main-content">
```

Comment:
I think you can use <main> to be your "main wrapper". You could move the class to it